tests/pkg_tweetnacl/Makefile

```diff
@@ -0,0 +1,17 @@
+include ../Makefile.tests_common
+
+BOARD_INSUFFICIENT_MEMORY := arduino-duemilanove \
+                             arduino-uno \
+                             arduino-mega2560 \
+                             mega-xplained \
+                             nucleo-f031k6 \
+                             #
```

Comment:
Just missing the `nucleo-f042k6` here to make `murdock` happy. You can inline squash it to test it directly so it is ready to merge after.

Reply:
> Just missing the `nucleo-f042k6` here to make `murdock` happy. You can inline squash it to test it directly so it is ready to merge after.

Fixed -_-